### PR TITLE
Editor fixes

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/modals/add-widgets/histogram/histogram-option-view.js [hunk 1]
```diff
@@ -63,7 +63,9 @@ module.exports = CoreView.extend({
           }));
           var stats = self.$('.js-histstats').children();
           self.$(stats[0]).text(graph.getNullsPercentage() + '% null');
-          self.$('.js-histstats').show();
+          self.$('.js-histstats').css('display', 'flex');
+        } else {
+          self.$('.js-Histogram').html("<div class='Widget-histogramFakeContainer'><p class='Widget-categoryFakeText'></p><div class='Widget-histogramFake'></div></div>");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/histogram/histogram-option-view.js]: Could we move this into a template please?

File: lib/assets/javascripts/cartodb3/components/modals/add-widgets/formula/formula-option-view.js [hunk 2]
```diff
@@ -69,7 +69,13 @@ module.exports = CoreView.extend({
             self.$(stats[0]).text(graph.getNullsPercentage() + '% null');
             self.$(stats[1]).text(aggregation.toFixed(2).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ','));
             self.$('.js-formulastats').show();
+          } else {
+            self.$('.js-formulastats').html("<div class='Widget-formulaFakeContainer'><p class='Widget-categoryFakeText'></p><div class='Widget-formulaFake'></div></div>");
+            self.$('.js-formulastats').show();
           }
+        } else {
+          self.$('.js-formulastats').html("<div class='Widget-formulaFakeContainer'><p class='Widget-categoryFakeText'></p><div class='Widget-formulaFake'></div></div>");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/formula/formula-option-view.js]: Same here, a template here would help.

File: lib/assets/javascripts/cartodb3/components/modals/add-widgets/formula/formula-option-view.js [hunk 3]
```diff
@@ -69,7 +69,13 @@ module.exports = CoreView.extend({
             self.$(stats[0]).text(graph.getNullsPercentage() + '% null');
             self.$(stats[1]).text(aggregation.toFixed(2).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ','));
             self.$('.js-formulastats').show();
+          } else {
+            self.$('.js-formulastats').html("<div class='Widget-formulaFakeContainer'><p class='Widget-categoryFakeText'></p><div class='Widget-formulaFake'></div></div>");
+            self.$('.js-formulastats').show();
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/formula/formula-option-view.js]: Dixit!

File: lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-options-view.js [hunk 4]
```diff
@@ -25,7 +25,9 @@ module.exports = CoreView.extend({
       className: 'WidgetList-item',
       model: m
     });
+    console.log(view);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-options-view.js]: Console!

File: lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-option-view.js [hunk 5]
```diff
@@ -63,7 +63,9 @@ module.exports = CoreView.extend({
           var stats = self.$('.js-catstats').children();
           self.$(stats[0]).text(graph.getNullsPercentage() + '% null');
           self.$(stats[1]).text((graph.getPercentageInTopCategories() * 100).toFixed(2) + _t('components.modals.add-widgets.percentage-in-top-cats'));
-          self.$('.js-catstats').show();
+          self.$('.js-catstats').css('display','flex');
+        } else {
+          self.$('.js-Category-bar').html("<p class='Widget-categoryFakeText'></p><div class='Widget-categoryFake'></div>");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-option-view.js]: Template!!

File: lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-options-view.js [hunk 6]
```diff
@@ -26,6 +26,7 @@ module.exports = CoreView.extend({
       model: m
     });
     this.addView(view);
+    this.$('.fake').remove();
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-options-view.js]: js-fake please :)

Comment (piensaenpixel) [inline lib/assets/javascripts/cartodb3/components/modals/add-widgets/category/category-options-view.js]: Sorry, really we don't need that. Fixing it.

**Pull request merged**